src/policy.h

```diff
@@ -39,6 +39,7 @@ struct qd_policy_denial_counts_s {
     int sessionDenied;
     int senderDenied;
     int receiverDenied;
+    int maxSizeMessagesDenied;
```

Comment:
In general don't we use unsigned values for counters elsewhere (like in the core code - all counters are uint64_t)